# Global Foodomics

This repository contains code to generate the figures for the manuscript **Reference data based insights expand understanding of human metabolomes** by Gauglitz et al.

> The human metabolome has remained largely unknown, with most studies annotating ~10% of features. In nucleic acid sequencing, annotating transcripts by source has proven essential for understanding gene function. Here we generalize this concept to stool, plasma, urine and other human metabolomes, discovering that food-based annotations increase the interpreted fraction of molecular features 7-fold, providing a general framework for expanding the interpretability of human metabolomic “dark matter.”

## Overview

- [Figure 1d](https://github.com/DorresteinLaboratory/GlobalFoodomics/blob/master/notebooks/fig1d_flow_food_type_hierarchy.ipynb): Summary of the spectra observed in plasma from a sleep deprivation study at hierarchical food ontology levels.
- [Figure 2](https://github.com/DorresteinLaboratory/GlobalFoodomics/blob/master/notebooks/fig2_spectral_interpretation_rate.Rmd): Increases of MS/MS spectral match rates when using interpretive metabolomics at the data set level.
- [Figure 3](https://github.com/DorresteinLaboratory/GlobalFoodomics/blob/master/notebooks/fig3_dietary_recommendation.ipynb): Using interpretive metabolomics to assess dietary recommendations at the study level.

## Contact

For more information, please contact [Pieter Dorrestein](mailto:pdorrestein@health.ucsd.edu).
